src/js/components/FileInput/stories/Custom.js

```diff
@@ -5,6 +5,19 @@ import { Trash } from 'grommet-icons';
 
 const customTheme = {
   fileInput: {
+    button: {
+      hover: {
+        color: 'pink',
```

Comment:
the `pink` color is too light and has a bad color contrast, can we change it to 'brand' or something that will be more clear?